### DISCUSSION TITLE
How to handle top_k in multi vector foreach key

### DISCUSSION THREAD

**Discussion by sandraplato1234** [milvus-io/milvus#4662] [Q&A and General discussion]
Hi ,

I want to know how to use milvus if i want to store multi vectors for the same key and support top_k for different keys , in case that more than one vector is of the same key, it will return as one key and need other k-1 keys to return
I uses Faiss as index
Thanks in advance

**Comment by miia12** [comment]
Sorry, I couldn't quite understand your question. With `id` Milvus can get vector by `get_entity_by_id`. However, this function cannot get every vector with the same id except for the latest inserted vector.

**Reply by sandraplato1234** [reply]
Thanks for your quick reply.
I want for each image store for example 5 augmentations and all these vectors  i want to index with the same id because they represent the same image.and when search for top_k only get once the same id even there are more than 1 of the same id that has high accurracy. is that possible?
Thanks
